tests/tests/smt_tests.cpp

```diff
@@ -0,0 +1,57 @@
+#ifdef IS_TEST_NET
+
+#include <boost/test/unit_test.hpp>
+
+#include <steemit/protocol/exceptions.hpp>
+
+#include <steemit/chain/database.hpp>
+#include <steemit/chain/database_exceptions.hpp>
+#include <steemit/chain/hardfork.hpp>
+#include <steemit/chain/steem_objects.hpp>
+
+#include "../common/database_fixture.hpp"
+
+using namespace steemit::chain;
+using namespace steemit::protocol;
+using fc::string;
+
+BOOST_FIXTURE_TEST_SUITE( smt_tests, clean_database_fixture )
+
+BOOST_AUTO_TEST_CASE( elevate_account )
```

Comment:
This does not match the test structure for other ops. Each op has a validate, authority, and apply unit test.

Claim reward balance is a good example of how this should be formatted.

[validate](https://github.com/steemit/steem/blob/master/tests/tests/operation_tests.cpp#L5781-L5834)
[authorities](https://github.com/steemit/steem/blob/master/tests/tests/operation_tests.cpp#L5836-L5859)
[apply](https://github.com/steemit/steem/blob/master/tests/tests/operation_tests.cpp#L6044-L6140)

Reply:
As this request is a blocker to further SMT development we'd like to take it over and implement it in dedicated `smt-develop` branch, that is already created and has the `1457-step-by-step` merged to. Do you mind us going for it?